lib/datastore/request.js

```diff
@@ -113,7 +113,7 @@ function DatastoreRequest() {}
  * @param {Key|Key[]} keys - Datastore key object(s).
  * @param {function} callback - The callback function.
  * @param {?error} callback.err - An error returned while making this request
- * @param {module:datastore/entity|module:datastore/entity[]} callback.entity -
+ * @param {object|object[]} callback.entity -
```

Comment:
We can probably change the description, maybe "The entity object(s) which match the provided keys."